app/decorators/course_decorator.rb

```diff
@@ -46,6 +46,68 @@ def funding
     end
   end
 
+  def subject_name
+    if object.subjects.count == 1
+      object.subjects.first.subject_name
+    else
+      object.name
+    end
+  end
+
+  def has_scholarship_and_bursary?
+    has_bursary? && has_scholarship?
+  end
+
+  def bursary_first_line_ending
+    if requirements.count > 1
+      ":"
+    else
+      "#{requirements.first}."
+    end
+  end
+
+  def requirements
```

Comment:
can we call this something else as
https://find-postgraduate-teacher-training.education.gov.uk/course/T92/X130#section-entry

on the course details page there is already something called `requirements`

maybe financial_incestive_requirements

Reply:
Uh I'm not so sure about `financial_incestive_requirements`...
... `financial_incentive_requirements` might be more family-friendly